Update ru.po

a1ex-ak opened the pull request:
Bringing the translation into line with the Gnome interface standard

a1ex-ak committed:
Update ru.po

Bringing the translation into line with the Gnome interface standard

ai commented on po/ru.po:
```diff
@@ -243,17 +243,17 @@ msgstr "Удалено"
 #: lib/thresholdPanel.js:406
 msgid "Max Lifespan"
-msgstr "Макс. срок"
+msgstr "Энергосберегающий"
```
I am not sure that it explain what it does.

It is not about “saving energy” (the direct translation of энергосберегающий) and extending battery lifespan.

We can try `Сберегающий` (saving) if we want to use adjective.

a1ex-ak replied:
> I am not sure that it explain what it does.
> 
> It is not about “saving energy” (the direct translation of энергосберегающий) and extending battery lifespan.
> 
> We can try `Сберегающий` (saving) if we want to use adjective.

I want the menu to be consistent with the Gnome translation concept, so let's use Сберегающий (saving).

<img width="406" height="575" alt="Снимок экрана от 2025-11-03 17-20-22" src="https://github.com/user-attachments/assets/210a383d-ac85-4cad-8485-b427077bdd8c" />

maniacx replied:
@a1ex-ak 
@ai 


Look at this comment by @BANOnotIT

<img width="863" height="257" alt="image" src="https://github.com/user-attachments/assets/8f2a02a5-9483-4767-874f-ce7a8ee509bf" />

https://github.com/maniacx/Battery-Health-Charging/pull/170#pullrequestreview-3411443163

**Looking at Gnome Settings Translation.**

Power saving (Reduce use of battery by using CPU power saver profile)
<img width="379" height="78" alt="image" src="https://github.com/user-attachments/assets/f556b41b-1626-4e44-8164-03062392002c" />


Limit charging
<img width="653" height="279" alt="image" src="https://github.com/user-attachments/assets/2d391ed9-12ca-4b8a-9161-a968e674c43f" />

a1ex-ak replied:
> @a1ex-ak @ai
> 
> Look at this comment by @BANOnotIT
> 
> <img alt="image" width="863" height="257" src="https://private-user-images.githubusercontent.com/4295343/509017450-8f2a02a5-9483-4767-874f-ce7a8ee509bf.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyMzkzMDIsIm5iZiI6MTc2MjIzOTAwMiwicGF0aCI6Ii80Mjk1MzQzLzUwOTAxNzQ1MC04ZjJhMDJhNS05NDgzLTQ3NjctODc0Zi1jZTdhOGVlNTA5YmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTExMDRUMDY1MDAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODVmYjc3MDA0MjBiNjkwZmYxMzVhYjQwYjRmOWRlNDI5ZGIxNmUzZmUxNWZlNzhhNzRmYjgxYTdiMzhlMGI1YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.4kcuO6N2L8qBmmhAOFcCEaAF-v2rv8TEuYvhC6AkHa8">
> [#170 (review)](https://github.com/maniacx/Battery-Health-Charging/pull/170#pullrequestreview-3411443163)
> 
> **Looking at Gnome Settings Translation.**
> 
> Power saving (Reduce use of battery by using CPU power saver profile) <img alt="image" width="379" height="78" src="https://private-user-images.githubusercontent.com/4295343/509019088-f556b41b-1626-4e44-8164-03062392002c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyMzkzMDIsIm5iZiI6MTc2MjIzOTAwMiwicGF0aCI6Ii80Mjk1MzQzLzUwOTAxOTA4OC1mNTU2YjQxYi0xNjI2LTRlNDQtODE2NC0wMzA2MjM5MjAwMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTExMDRUMDY1MDAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjdlMjc5ZTUxMTIxM2IzZDdiYTUzMzFiZTY2OGNkMTAzNzY0ZjcyMzI3YjhlNThkOGZjOTE3NDU1ZDk0NDhmNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.9us8Suc6M-ZuoUBt9baCHb2ygrlOWq2VSVG1CK50Zmk">
> 
> Limit charging <img alt="image" width="653" height="279" src="https://private-user-images.githubusercontent.com/4295343/509019703-2d391ed9-12ca-4b8a-9161-a968e674c43f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyMzkzMDIsIm5iZiI6MTc2MjIzOTAwMiwicGF0aCI6Ii80Mjk1MzQzLzUwOTAxOTcwMy0yZDM5MWVkOS0xMmNhLTRiOGEtOTE2MS1hOTY4ZTY3NGM0M2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTExMDRUMDY1MDAyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTQ1ZmE5ODQ3ZmJhZDEyYjJlZTc5M2FjNGY2NGFmYzM1YzAwYjA1OGZjZWMwMTE5YWU1NzBhNzYwZWY5MTJmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.DoqzKSrqf_qutaEjFt-c8zrnR51SngYAWCCKBWYHNsU">

@maniacx @ai 
Corrected with your wishes

ai commented on po/ru.po:
```diff
@@ -243,17 +243,17 @@ msgstr "Удалено"
 #: lib/thresholdPanel.js:400
 msgid "Full Capacity"
-msgstr "Полный"
+msgstr "Производительный"
```
`Производительный` means performance. What about `Максимальный` to fit thew GNOME style?

a1ex-ak replied:
Максимальный - doesn't quite convey the meaning

ai replied:
Why? (we can switch to Russian here to discuss meaning nuances)

a1ex-ak replied:
> Why? (we can switch to Russian here to discuss meaning nuances)

да согласен, на русский )

Опять же я исхожу из общей концепции перевода Gnome и в нем для любых режимов работы (от батареи или от силового провода) есть три режима:
- Производительный
- Сбалансированный
- Энергосберегающий

Поразмыслив, в нашем случае давайте сделаем как Вы предлагаете:
- Максимальный
- Сбалансированный
- Сберегающий или Максимальная служба (чтобы не повторялись буквы С)

ai replied:
«Максимальная служба» будет конфликтовать с «Максимальный».

Я думаю Максимальный/Сбалансированный/Сберегающий — будет хорошо.

a1ex-ak replied:
> «Максимальная служба» будет конфликтовать с «Максимальный».
> 
> Я думаю Максимальный/Сбалансированный/Сберегающий — будет хорошо.

Отлично, пусть будет так )

a1ex-ak replied:
@ai 
Финалим?

ai replied:
Ага, обновляй везде (включая там где в @BANOnotIT предлагал)

a1ex-ak commented:
@ai hello
Can you correct what was written above?
I think it would be better to say "Full Capacity Charge Mode Set," "Balanced Charge Mode Set," etc.
Я думаю, что лучше - "Установлен режим заряда Полная ёмкость", "Установлен режим заряда Сбалансированный" и т.д. )

a1ex-ak commented:
@maniacx @ai 
I corrected the wishes)

ai commented:
> I corrected the wishes)

There is still `Энекргосберегающий` in PR. Maybe you forgot to push?

a1ex-ak commented:
> > I corrected the wishes)
> 
> There is still `Энекргосберегающий` in PR. Maybe you forgot to push?

I think I've corrected everything and pressed it) Would you like to try it?
